infra/base-images/base-runner/bad_build_check

```diff
@@ -433,6 +466,13 @@ function main {
   result=$?
   checks_failed=$(( $checks_failed + $result ))
 
+  if [ -n "$MORE_BAD_BUILD_CHECKS" ]
```

Comment:
I think this may still break due to the `-u` argument passed to the hashbang bash command.

You'll need something like `${MORE_BAD_BUILD_CHECKS:-}` instead